pkg/newcache/newcache.go

```diff
@@ -0,0 +1,222 @@
+/*
+Copyright 2016 The Fission Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package newcache implements a simple cache implementation having one value mapped by two keys.
+// As of now this package is only used by poolmanager executor
+package newcache
```

Comment:
Can we use a better name instead of `newcache` please?

Reply:
Done